wiki/markdown/docker.md

```diff
@@ -0,0 +1,68 @@
+# Docker
+
+In Eiffel-Intelligence frontend source code a Dockerfile is provided which helps the developer or user to build the local Eiffel-Intellegence frontend source code changes to a Docker image.
+With the Docker image user can tryout the Eiffel-Intelligence frontend on a Docker Host or in a Kubernetes cluster.
```

Comment:
Same comments here as on the EI back-end repo documentation :) I'm guessing the files are duplicated?

I think it sounds more natural to have "source code repository" in the beginning. Also the spelling of "tryout" bothers me for some reason. Not sure if "try-out", "try out" or just "try" is better.

Reply:
Fixed.